src/modules/board.js

```diff
@@ -0,0 +1,32 @@
+import { postData, getData } from './appi';
+
+function Score(score, name) {
+  this.score = score;
+  this.user = name;
+}
```

Comment:
- Hey, great job with this function, however, according to the project requirement, let's make this an arrow function since we're using the ES6 syntax. please, kindly update this function.